docs/user/setup/README.md

```diff
@@ -180,6 +181,96 @@ Alternatively, you can use the native Kubernetes cluster provided by Docker Desk
 
 > :warning: The rest of the tutorial assumes a minikube cluster, however.
 
+### 3.1. Windows with Ubuntu WSL
+
+Follow these steps when running Ubuntu in WSL (tested on Ubuntu 22.04/24.04):
+
+
+- Install kubectl
+```bash
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
+chmod +x ./kubectl
+sudo mv ./kubectl /usr/local/bin/kubectl
+kubectl version
+```
+- Install Docker
```

Comment:
Just a question, what if docker Desktop for Windows is already installed? Then these steps would be somehow obsolete? since the docker daemon should be exposed to WSL?